test_base_fileurl_field/models/res_partner.py

```diff
@@ -1,44 +1,57 @@
 # Copyright 2019 Camptocamp SA
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
-from odoo import models, fields, api, _
+from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
 
+# make pytest happy
+# in pytest context module dependencies are not loaded
+# thus FileURL is unknown
+from odoo.addons import base_fileurl_field  # noqa
```

Comment:
we should import the field directly instead of relying on a patch (as we do for base_sparse_field) :wink: 
```suggestion
from odoo.addons.base_fileurl_field.fields import FileURL
```

Reply:
IMO we should be able to use it from `odoo.fields` from any field Type registered in `odoo.fields`.

`fields.MyFieldType` is a common usage and making exception is not great.

Reply:
IMO is just a dirty hack. In fact, it's a patch and you have no certainty that is going to be loaded at the right time.
Plus, no framework out there - to my knowledge relies on such approach and just imports the thing you need from where it stays.
Not to mention that is way more explicit :wink: